roles/aap_setup_install/tasks/main.yml

```diff
@@ -11,7 +11,8 @@
     url: "https://{{ controller_hostname }}/"
     method: GET
     user: admin
-    password: "{{ aap_setup_prep_inv_vars.all.admin_password }}"
+    password: "{{ aap_setup_prep_inv_secrets.all.admin_password | default(false) or
```

Comment:
Is this right? I assume you want to fallback to `aap_setup_prep_inv_vars.all.admin_password` but my reading of this is that it will just default(false) which is presumably not what we want to do here?

Is the correct solution here not:

```
    password: "{{ aap_setup_prep_inv_secrets.all.admin_password | default(aap_setup_prep_inv_vars.all.admin_password) }}"
```

Reply:
yup, that was an artifact of me trying something else, defaulting to `aap_setup_prep_inv_vars.all.admin_password` is the way.